lib/importer/factory/object_factory.rb

```diff
@@ -64,23 +66,21 @@ def search_by_identifier
       def create
         attrs = create_attributes
         @object = klass.new
-        run_callbacks :save do
-          run_callbacks :create do
-            klass == Collection ? create_collection(attrs) : work_actor.create(environment(attrs))
-          end
-        end
-        log_created(object)
+        klass == Collection ? create_collection(attrs) : work_actor.create(environment(attrs))
+        attach_file_to_work
       end
 
-      def log_created(obj)
-        msg = "Created #{klass.model_name.human} #{obj.id}"
-        Rails.logger.info("#{msg} (#{Array(attributes[system_identifier_field]).first})")
-      end
-
-      def log_updated(obj)
-        msg = "Updated #{klass.model_name.human} #{obj.id}"
-        Rails.logger.info("#{msg} (#{Array(attributes[system_identifier_field]).first})")
-      end
+      ## below methods are commented out to pass rubocop inspection ("Class has too many line");
```

Comment:
Rather than commenting out the logging feature, how about changing the threshold in `.rubocop_todo.yml`?

Reply:
It didn't occur to me to change the enforced practice - will do thanks!